### ISSUE TITLE
Test collection of embeddables inside an `@Embeddable`

### ISSUE THREAD

**Issue by blafond** [hibernate/hibernate-reactive#533]
Example:
```
	@Embeddable
	public static class Phone {
                 ...
                 @ElementCollection
		 private List<AlternativePhone> alternatives = new ArrayList<>();
       }

	@Embeddable
	public static class AlternativePhone {
                 ...
       }
```

This should already work because of #44 but tests are missing.

**Comment by DavideD** [comment]
This is done, isn't it?

**Comment by gavinking** [comment]
Folks I think the problem is that putting `[#533]` doesn't work: it doesn't create a link to the issue from the PR. You need to put something like `fixes #533` in the description of the pull request. Otherwise the issue won't get automatically closed when you merge the PR.
